db/seeds.rb

```diff
@@ -34,225 +106,207 @@
-    item_type: 'ingredient',
-    description: Faker::Food.description,
-    expiration_date: Faker::Date.between(from: 2.days.from_now, to: 5.days.from_now)
-  )
-end
-puts "#{'5'.blue} ingredient items created for #{user.username.light_cyan} "
-puts '--------------------------------------------------------------------'.light_black
+puts " Demo persona: #{@shayna.username.light_cyan} has been created with a #{shaynas_meal.name.cyan} meal"
+add_allergens_and_diets(shaynas_meal)
 
-#############################################################################
-#-------------------------SEED DB WITH MEAL ITEMS---------------------------#
-#-------!These items DO NOT have any allergens or dietary restrictions!-----#
-#############################################################################
-
-# Creates users with meal items to fill DB.
-user = User.create!(
-  email: Faker::Internet.email,
-  username: Faker::Name.first_name + Faker::Creature::Dog.name,
+# Demo user Williams will be the RECEIVER that justin gives a meal to.
+@williams = User.create!(
+  email: 'williams@foodfor.all',
+  username: 'Williams',
   password: '123456',
-  address: locations.sample
+  address: '5305 drolet st, montreal'
 )
-puts "#{'✓'.light_green} Demo user: #{user.username.light_cyan} has been created."
-
-5.times do
-  meal = Item.create!(
-    user_id: user.id,
-    name: Faker::Food.dish,
-    status: @status_list.sample,
-    item_type: 'meal',
-    description: Faker::Food.description,
-    expiration_date: Faker::Date.between(from: 2.days.from_now, to: 5.days.from_now)
-  )
-end
-puts "#{'5'.blue} meal items created for #{user.username.light_cyan} "
-puts '--------------------------------------------------------------------'.light_black
+puts " Demo persona: #{@williams.username.light_cyan} has been created to rent #{@shayna.username.light_cyan}'s #{shaynas_meal.name.cyan} meal"
+puts "#{'✓ Personas '.light_green}created"
+puts ''
 
 #############################################################################
-#-------------------------SEED DB WITH MEAL ITEMS---------------------------#
-#----------------------------!WITH ALLERGENS!-------------------------------#
+#----------------------------SEED DB WITH USERS-----------------------------#
 #############################################################################
+puts '------------------------Users with meals----------------------------'.light_black
 
-# Creates users with meal items to fill DB.
-user = User.create!(
-  email: Faker::Internet.email,
-  username: Faker::Name.first_name + Faker::Creature::Dog.name,
-  password: '123456',
-  address: locations.sample
-)
-puts "#{'✓'.light_green} Demo user: #{user.username.light_cyan} has been created."
-
-5.times do
+# Creates users, each user will have a meal to give.
+@counter_from_zero = 0
+7.times do
+  user = User.create!(
+    email: Faker::Internet.email,
+    username: (Faker::Name.first_name + Faker::Name.last_name).capitalize,
```

Comment:
@DenzelBraithwaite Do you think it's possible to add space (+ " " +) between first name and last name? Because now the sample names have no space (see attached). Not sure if this is doable.
![Screenshot 2022-05-31 at 3 31 43 PM](https://user-images.githubusercontent.com/75033073/171288481-e48513e0-c8ef-45ff-b50a-4d9c49eb4ea4.png)